CHANGELOG.txt

```diff
@@ -190,6 +201,13 @@
             - A class 'Assert()', which factorizes some validation code, has been added.
             - A builtin method 'instance' has been added in 'creme.utils.newJQueryPlugin'.
         # Apps :
+            * CremeCore :
```

Comment:
Again: there is never "CremeCore" section in "Apps" section (because it would not mean anything, all code is is an app).
Line 189 replace

`# A custom manager has been added to CustomField ; some queries are now cached.`

by

```
# In 'creme_core.models': 
     - A custom manager has been added to CustomField ; some queries are now cached.
    - Helpers have been added SettingValue manager:
         - SettingValue.objects.exists_4_key()
        - SettingValue.objects.value_4_key()
        - SettingValue.objects.set_4_key()
```

Line 194, in the section `"# In unit tests :"` add this:

`- New with-statement OverrideSettingValueContext which allows local changes in SettingValues.`

After the section ` "# In 'creme_core.core.entity_cell.EntityCell'"` add this 

`# In "creme_core.core.setting_key',  the description of a SettingKey can now be multiline.`